docs/deep-dive/project-structure.md

```diff
@@ -8,54 +8,36 @@ In this article we'll look at the project structure and explain the responsibili
 
 Once your [project is set up](/docs/get-started/quick-start.md), it will look like this:
 
-```
-new-project
-├── api/
-│   ├── .serverless/
-│   ├── .env.json
-│   └── serverless.yml
-├── apps/
-│   ├── .serverless/
-│   ├── admin/
-│   ├── site/
-│   └── serverless.yml
-├── packages/
-│   ├── project-utils/
-│   └── theme/
-├── .env.json
-├── .gitignore
-├── .prettierrc.js
-├── README.md
-├── package.json
-└── webiny.js
-```
+![New Project Folder Structure](/img/deep-dive/project-structure/folder-structure.png)
```

Comment:
I prefer markdown tree version for project structure like we currently have

What do you think @webiny/webinydevteam ?

Reply:
I think in some cases that might cause a lot of typing if we have lots of files to show unless we can find a plugin for vs code or something that can export that for us automatically.

Reply:
you an do it easily using the `tree` command. Built-in on Linux, installable using `brew` on OSX, maybe even works on Windows in `git bash`. 

Simply run ` tree . -L 2` to create a tree structure:
```
.
├── api
│   ├── apolloGateway
│   ├── databaseProxy
│   ├── example.env.json
│   ├── files
│   ├── formBuilder
│   ├── headless
│   ├── i18n
│   ├── pageBuilder
│   ├── resources.js
│   ├── security
│   ├── services
│   └── webiny.config.js
├── apps
│   ├── admin
│   ├── databaseProxy
│   ├── future
│   ├── resources.js
│   └── site
├── example.env.json
└── webiny.root.js
```

Reply:
@EmilK15 you're using WSL so you can easily use this.

Reply:
Added the tree-view. Also updated a few things I noticed about `apps` and `webiny.config.js` locations for users to edit.